src/metabase/api/app.clj

```diff
@@ -225,8 +226,8 @@
                                                             {"type" "link"
                                                              "linkType" "page"
                                                              "parameterMapping" {(str "scaffold_" table-id) {"source" {"type" "column",
-                                                                                                                       "id" "ID",
-                                                                                                                       "name" "ID"},
+                                                                                                                       "id" (:name pk-field)
```

Comment:
Could/should we use the `:id` as ID?

Reply:
That doesn't seem to be what the front end does, and we have to follow them.